metadata-ingestion/scripts/datahub_preflight.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+echo "Not running anymore preflight script"
+exit
```

Comment:
Should we just delete this script entirely? We can always recover it from git history